css/style.css

```diff
@@ -0,0 +1,285 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #f4f5f6;
+}
+
+.container-fluid {
+  min-width:  100%;
+  padding:0 !important;
+}
+
+header {
+  min-width: 100%;
+  margin-top: 70px;
+}
+
+.header-main-3-lg {
+  position: fixed;
+  min-height: 20px;
+  display: none;
+  z-index: 10;
+}
+
+header > div {
+  background-color: #f72210;
+}
+
+.header-btn div {
+ margin-bottom: 20px;
+}
+
+header div:first-child {
+ position: relative;
+}
+
+.header-date,
+.header-btn {
+ padding-top: 30px;
+}
+
+header svg {
+ transform: scale(2.3);
+}
+
+.login {
+ font-size: 16px;
+ padding-right: 16px;
+ font-weight: bold;
+}
+
+.subscribe {
+ border-radius: 15px;
+ padding: 8px;
+ background-color: #ffe400;
+ margin-right: 30px;
+}
+
+.hr-icon-bottom {
+ position: absolute;
+ top: 100%;
+ right: 0;
+ content: "";
+ display: block;
+ width: 92px;
+ height: 14px;
+ background-color: #f72210;
+ border-left: 22px solid white;
+ border-top: 16px solid #f72210;
+}
+
+.header-main-2 img,
+.header-main-3 img {
+ width: 50px;
+ height: 50px;
+}
+
+.header-top svg {
+ display: inline;
+ width: 236px;
+ height: 36px;
+}
+.main2-svg-link {
+ margin-top: 0 !important;
+ width: 100px !important;
+ height: 20px;
+ padding: 0 !important;
+}
```

Comment:
- [ ] Try not to use`!important` if it can  be avoided. this can be done with the following bootstrap classes:
-  margin top : `mt-0`
- padding: `p-0`